dataprofiler/profilers/profile_builder.py

```diff
@@ -397,7 +400,7 @@ def clean_data_and_get_base_stats(df_series, sample_size,
         na_columns = dict()
         true_sample_list = set()
         total_sample_size = 0
-        query = '|'.join(null_values_and_flags.keys())
+        query = '|'.join(self._null_values.keys())
```

Comment:
This is the only place _null_values are accessed, and it only needs the keys. What are the values for/how are they supposed to be used?